### DISCUSSION TITLE
Issue with not-equals sign

### DISCUSSION THREAD

**Discussion by mspeculatrix** [TriliumNext/Trilium#3579] [Q&A]
Whenever I enter the 'not equals' operator (!=) Trilium changes it to a not-equal symbol (an equals sign with a slash through it). This is irritating when making codes notes. Is there any way of switching off this behaviour? Or a workaround?

**Comment by zadam** [comment]
This is called "font ligature" and is actually not a behavior of trilium, but of the font.

Try changing the font for Code:


<img width="624" alt="image" src="https://user-images.githubusercontent.com/617641/215901283-71430aeb-d79d-4680-be6c-5ae7ffcdd978.png">
